src/component/splash/SplashPageEnvironments/SplashPageEnvironments.tsx

```diff
@@ -1,196 +0,0 @@
-import { SplashPageEnvironmentsContent } from 'component/splash/SplashPageEnvironments/SplashPageEnvironmentsContent/SplashPageEnvironmentsContent';
-import { SplashPageEnvironmentsContainer } from 'component/splash/SplashPageEnvironments/SplashPageEnvironmentsContainer/SplashPageEnvironmentsContainer';
-import { VpnKey, CloudCircle } from '@mui/icons-material';
-import { useStyles } from 'component/splash/SplashPageEnvironments/SplashPageEnvironments.styles';
-import { ReactComponent as Logo1 } from 'assets/img/splashEnv1.svg';
-import { ReactComponent as Logo2 } from 'assets/img/splashEnv2.svg';
```

Comment:
Could delete these images well. Same for `envSplash2.png` in the same dir, looks unused.